elastic/kibana
# [Uptime] [Alerting] adjust Uptime RAC RBAC and index settings

Fixes #108202

## Summary

This PR adjusts Uptime RBAC settings. Previously, we were aliasing uptime RAC data with the "synthetics" name. However, synthetics does not match with the "uptime" capability within our permission model. To keep things simple, we should continue using "uptime" for alerting related items.

### Testing
1. Start heartbeat with a monitor that will always resolve to down
2. Navigate to Uptime and create a monitor status alert by either using the alert status toggle or alerts and rules dropdown in the heading
3. Navigate to `/app/observability/alerts` to observe the alert
